lib/msf/ui/console/command_dispatcher/db/klist.rb

```diff
@@ -76,7 +78,16 @@ def cmd_klist(*args)
 
     if mode == :delete
       result = kerberos_ticket_storage.delete_tickets(ids: ticket_results.map(&:id))
-      delete_count = result.size
+      entries_affected = result.size
+    end
+
+    if mode == :activate || mode == :deactivate
+      result = set_activation_status(mode, ticket_results)
+      entries_affected = result.size
+      # Update the contents of ticket results to display the updated status values
+      # TODO: should be able to use the results returned from updating loot
+      # but it returns a base 64'd data field which breaks when bindata tries to parse it as a ccache
```

Comment:
Is this just for the remote db? That shouldn't be happening 👀

Reply:
It happened when I was using the remote DB, I would assume it's only a problem there but I haven't checked yet